bokehjs/src/lib/models/glyphs/defs.ts

```diff
@@ -405,6 +426,25 @@ function square_x(ctx: Context2d, i: number, r: number, visuals: VectorVisuals):
   }
 }
 
+function star(ctx: Context2d, i: number, r: number, visuals: VectorVisuals): void {
+  _one_star(ctx, r)
```

Comment:
I don't think it's necessary to put the drawing code into a separate `_one_star` function unless there is an expectation that it will be re-used/combined with other functions in additional glyphs now or in the future (seems unlikely?)